### DISCUSSION TITLE
Use display name in follow up

### DISCUSSION THREAD

**Discussion by Milo123459** [lowlighter/metrics#460] [Ideas]
![image](https://user-images.githubusercontent.com/50248166/128665619-e5ef480d-2d04-425e-9ef3-fb9126f3534a.jpeg)

In my opinion it should show display name rather than username￼, for example mine is Milo so Issues created by Milo

**Comment by lowlighter** [comment]
Yes, this could be changed quite easily 👍

**Comment by Nixinova** [comment]
Implemented in #465
